automation/routine_publish_signals.go

```diff
@@ -189,9 +133,12 @@ func (p PublishSignals) Do(ctx context.Context, c *clarify.Client, opts PublishO
 }
 
 // addItems adds items that require update to dest from all signals matching
-// the integration ID and query q.
-func (p PublishSignals) addItems(ctx context.Context, dest map[string]views.ItemSave, c *clarify.Client, integrationID string, q params.ResourceQuery, opts PublishOptions) (bool, error) {
-	results, err := c.Admin().SelectSignals(integrationID, q).Include("item").Do(ctx)
+// the integration ID and query.
+func (p PublishSignals) addItems(ctx context.Context, cfg *Config, dest map[string]views.ItemSave, integrationID string, query fields.ResourceQuery) (bool, error) {
```

Comment:
nit: too many arguments

Reply:
I know, but as it's an internal method, i don't care.